Misc/NEWS.d/next/Core and Builtins/2023-05-08-10-34-55.gh-issue-104263.ctHWI8.rst

```diff
@@ -3,4 +3,4 @@ the meaning of the signalling / quiet bit is inverted from its usual
 meaning. Also introduce a new macro ``Py_INFINITY`` matching C99's
 ``INFINITY``, and refactor internals to rely on C99's ``NAN`` and
 ``INFINITY`` macros instead of hard-coding bit patterns for infinities and
-NaNs.
+NaNs. Thanks Sebastien Berg.
```

Comment:
Thanks Mark!  Slight oops, "Sebastian" :).

Reply:
Gah. Will fix. Sorry about that.

Reply:
Fixed in e8c36a224b4d223a22e67c76cd2419c633fd6e6d.